packages/compass-components/src/components/accordion.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { expect } from 'chai';
+
+import { fireEvent, render, screen, cleanup } from '@testing-library/react';
+
+import Accordion from './accordion';
+
+function renderAccordion() {
+  return render(
+    <Accordion
+      dataTestId="my-test-id"
+      text='Accordion Test'
+    >
+      <h1>Hello World</h1>
+    </Accordion>
+
+  );
+}
+
+describe('Accordion Component', function () {
+  
+  beforeEach(function () {
+  
```

Comment:
This `beforeEach` looks empty, strange that `eslint` didn't complain about it :/